lang_GENERIC/analyze/Naming_AST.ml

```diff
@@ -391,10 +406,10 @@ let resolve lang prog =
         (* note that some languages such as Python do not have VarDef 
          * construct
          * todo? should add those somewhere instead of in_lvalue detection? *)
-        VarDef ({ vinit = vinit; _}) when is_local_or_global_ctx env ->
+        VarDef ({ vinit = vinit; vtype = vtype }) when is_local_or_global_ctx env ->
```

Comment:
in theory you can use the shorter { vinit; vtype } pattern, which is equivalent to {vinit = vinit; vtype = vtype}
It's a recent feature of OCaml (not in SML I think).